examples/hosts/app-integration/external-data/README.md

```diff
@@ -24,6 +77,22 @@ TODO: Document the bot pattern and how it is surfaced in the example code.
 
 TBD
 
+### Concepts of data in this repository
+
+A few useful concepts to understand in implementing conflict resolution in this app:
+
+Task - This is the unit that can be edited and attributed to an author. It is also the level at which conflict resolution currently takes place. The task holds knowledge of the local edits as well as the external edits and can display one or both to the screen.
+
+TaskList - This can be compared to a "board" that holds all of the tasks. It is the larger visible entity to show up in a component within the Loop app, or any similar app. The app stores two types of TaskList's - a "draft" version and a "saved version". More on these below.
```

Comment:
I probably wouldn't refer to Loop specifically here. I would just refer to "the application" in the abstract.

```suggestion
TaskList - This can be compared to a "board" that holds all of the tasks. It is the larger visible entity to show up in a component within the Loop app, or any similar app. The app stores two types of TaskList's - a "draft" version and a "saved version". More on these below.
```